examples/sph/run_sph_taylor_green_vortex.py

```
@@ -185,5 +185,5 @@ def plot():
     model.evolve_until(i * dt_stop)
 
     # Dump name is "dump_xxxx.sham" where xxxx is the timestep
-    model.do_vtk_dump(dump_folder + "/dump_{:04}.vtk".format(i), True)
+    model.do_vtk_dump(dump_folder + f"/dump_{i:04}.vtk", True)
```

Ruff (0.16.0)

[warning] Boolean positional value in function call

(FBT003)